tap_hubspot/hubspot.py

```diff
@@ -762,7 +777,7 @@ def get_records(
                 "company_properties",
                 "archived_contacts",
                 "archived_companies",
-                "archived_deals",
+                "archived_deals"
```

Comment:
remember to format the code. First check if your formatter is `black`. Then when you do cmd+s, Your VScode should auto format on save. If your formatter doesn't work, you can always run `black .` command in the terminal to format your code.
<img width="1077" alt="Screenshot 2023-02-23 at 09 45 53" src="https://user-images.githubusercontent.com/48968138/220860127-813e3724-6925-421a-ace8-1880b6abd94f.png">